### ISSUE TITLE
Cross entropy

### ISSUE THREAD

**Issue by godardt** [EKami/planet-amazon-deforestation#4]
Use cross-entropy on all weather labels of the training data (add the None label to those who don’t have any). 
The labels concerned are: clear, partly cloudy, cloudy, haze
